### ISSUE TITLE
Не устанавливается на 4 версию.

### ISSUE THREAD

**Issue by kruatech** [LiveTex/Node-Pg#16]
![1444804081047](https://cloud.githubusercontent.com/assets/3449099/10476384/b4e5b1a8-7266-11e5-9a3b-9fa0d165887d.jpg)

**Comment by valeriakononenko** [comment]
node-gyp установлен?

**Comment by x2es** [comment]
попробуйте `npm install -g npm`
на новом `npm@3.x.x` скорее всего не будет такой проблемы

**Comment by hobbit-vt** [comment]
А это вся ошибка? Выше скрина ничего не было?

**Comment by kruatech** [comment]
node-gyp - установлен.
после npm install -g npm снова ошибка, но немног другая.
![111](https://cloud.githubusercontent.com/assets/3449099/10504373/53f9e676-7319-11e5-94f7-e4a02665b869.jpg)

**Comment by kruatech** [comment]
Выше скрина ничего не было.
Версия v4.2.1

**Comment by kop-kop** [comment]
Точно такая же проблема.
NodeJS 4.2.1
npm 3.3.8

**Comment by lexin4ever** [comment]
Скорее всего поможет даунгрейд Node.js до версии 0.10.36. У меня на v4.2.1 тоже не ставится.

**Comment by hobbit-vt** [comment]
Да, Joyent в версии _0.12.x_ обновило _v8_, в котором поменяли API для native extensions. Соответсвенно работает только в версии _0.10.x_.

**Comment by kruatech** [comment]
Под 4 версию драйвер делаться не будет?

**Comment by cevek** [comment]
уже node 6 вышла

**Comment by freeneutron** [comment]
У меня Centos v7-64, Node v4.2.6 и такая ошибка:
![nimbus-image-1463801720773](https://cloud.githubusercontent.com/assets/1611018/15446154/24d01248-1f40-11e6-9636-86bddcb351ed.png)

**Comment by valeriakononenko** [comment]
> > make: g++: Команда не найдена

Соответственно, надо его установить

**Comment by freeneutron** [comment]
Установил `yum install gcc-c++`. Теперь такая ошибка: 
![nimbus-image-1463844469432](https://cloud.githubusercontent.com/assets/1611018/15449301/ee21d178-1fa3-11e6-90ee-fbc8b841e987.png)

**Comment by lexin4ever** [comment]
В readme указана зависимость от библиотек `libpq` и `libjemalloc`. По логам не трудно догадаться что как минимум одна не установлена

**Comment by freeneutron** [comment]
Теперь такая проблема:
![nimbus-image-1463848725758](https://cloud.githubusercontent.com/assets/1611018/15449633/50b16fca-1fad-11e6-854c-dbc3fd71f4a5.png)

**Comment by hobbit-vt** [comment]
Этот проект компилируется только для Node 0.10.x.

**Comment by im-kulikov** [comment]
@hobbit-vt дальнейшее развитие/поддержка планируется? или можно форкать и дорабатывать для себя?

**Comment by hobbit-vt** [comment]
@im-kulikov к сожалению, в дальнейшем развитии проекта мы не заинтересованы =(

**Comment by im-kulikov** [comment]
@hobbit-vt понял, благодарю за ответ
